src/constants.ts

```diff
@@ -11,10 +11,11 @@ export const VBANK_RESERVE_ACCOUNT = 'vbank/reserve';
 export const GRAPH_DAYS = 90;
 
 export const SUBQUERY_URL = 'https://api.subquery.network/sq/agoric-labs/agoric-mainnet-v2';
+export const SUBQUERY_STAGING_URL = 'https://api.subquery.network/sq/agoric-labs/agoric-mainnet-v2__YWdvc';
```

Comment:
`SUBQUERY_STAGING_URL` seems similar to `SUBQUERY_URL` except the last part. Why use `STAGING` in `SUBQUERY_STAGING_URL`?

Reply:
as far as im aware, we have two subql deployments. the one marked as STAGING here is sort of a testing deployment that is indexed before the `SUBQUERY_URL` one is made